src/Layout.ts

```diff
@@ -236,8 +236,12 @@ export default class Layout<T extends FieldData = FieldData, U extends GenericPo
         params : ListParams<T, U> = {},
         ignoreEmptyResult = false
     ) : Promise<GetResponse<T, U>> {
+        // convert any boolean "omit" values in the query in the query to "true" or "false" strings
```

Comment:
That comment is quite redundant I'd say, think you can get rid of it :)

Reply:
Done. I think it was explaining why before, but got trimmed out at some point